addons/mail/static/src/messaging/entity/thread/thread.js

```diff
@@ -44,6 +44,9 @@ function ThreadFactory({ Entity }) {
             if ('mass_mailing' in data) {
                 data2.mass_mailing = data.mass_mailing;
             }
+            if ('model' in data) {
+                data2.model = data.model;
+            }
```

Comment:
that's wrong, `channel_info()` never passes `model`, so it should not be in convert

Reply:
@seb-odoo Seems that it's not wanted to do that in `convertData`

Reply:
(cf discussion below, I resolve this one)